book/tools.md

```diff
@@ -1,67 +1,96 @@
 # Tools
 
-This section points you to useful tools that support reproducible analysis.
+This section highlights tools that support reproducible analysis and research.
+This includes tools for general software development and bespoke packages that have been developed for government analysis.
+Those developed or contributed to within government are marked with an asterisk (*).
```

Comment:
Not all screen readers will read out `*`, so this should be changed for accessibility reasons

Reply:
Asterisk `*` replaced with `(gov)`, which should be more accessible and explicit. Feel free to change if you have an alternative suggestions